content/pages/setting-up-a-github-pages-site-with-jekyll/testing-your-github-pages-site-locally-with-jekyll.md

```diff
@@ -50,6 +50,10 @@ Before you can use Jekyll to test a site, you must:
   ```
 3. To preview your site, in your web browser, navigate to `http://localhost:4000`.
 
+{% note %}
+**Note**: If you are using Ruby 3.0 and Jekyll v4.2.x or older, you will need to add the gem "webrick" to your project's Gemfile and run `bundle install`
+{% endnote %}
+
```

Comment:
🎉 Great work!

Looking at the [article preview](https://docs-preview-16137-f59659.eastus.azurecontainer.io/en/pages/setting-up-a-github-pages-site-with-jekyll/testing-your-github-pages-site-locally-with-jekyll), I can see the "Note:" isn't in bold (the liquid note tags need a line before and after the text, see the [content-markup-reference](https://github.com/github/docs/blob/main/contributing/content-markup-reference.md#callout-tags) for an example) and the colon should be within the bolded part. 

Also, let's add the name of the gem before it's mention, and format the gem name in backticks (to be consistent with how we've named gems in other articles).

Since we're not including this step in the actual procedure above, where possible we should also clarify when a user needs to perform this step as part of the above procedure:

```suggestion
{% note %}

**Note:** If you are using Ruby 3.0 and Jekyll 4.2.x or older, you will need to add the `webrick` gem to your project's Gemfile prior to running `bundle install`.

{% endnote %}

```

Reply:
Okay right. Thank you, I'll get this resolved.